docs/testing.md

```diff
@@ -323,6 +323,10 @@ class UsersController < ApplicationController
   def index
     @user = authorized(User.all)
   end
+
+  def favorite
+    authorized_scope(User.all, context: {favorite: true})
+  end
```

Comment:
A context is usually a record (it's an acting _subject_); it's unlikely a bool. Let's change this example:

```suggestion
  def for_user
    user = User.find(params[:id])
    authorized_scope(User.all, context: {user:})
  end
```